dask/array/tests/test_stats.py

```diff
@@ -75,8 +75,10 @@ def test_one(kind):
     ],
 )
 def test_two(kind, kwargs):
+    # The sums of observed and expected frequencies must match
     a = np.random.random(size=30)
-    b = np.random.random(size=30)
+    b = a[::-1]
```

Comment:
This felt like the easiest way to make sure that they all match.

Reply:
Seems reasonable -  `np.random.permutation` should also work, but I don't think it provides any better coverage.